### PR TITLE
POL-458 AWS 1.6 Ensure hardware MFA is enabled for the 'root' user account

### PR DESCRIPTION
**Pull request by XOmniverse**
### Description

AWS 1.6 Ensure hardware MFA is enabled for the 'root' user account

### Issues Resolved

[List any existing issues this PR resolves]

### Contribution Check List

- [ ] All tests pass.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [ ] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

**Comment by XOmniverse** [comment]
Set to "DO NOT MERGE" since I can't properly test it against hardware MFA being enabled. This is for code review.

File: security/aws/iam_hwmfa_enabled_for_root/aws_iam_hwmfa_enabled.pt [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+  version: "1.0",
+  provider: "AWS",
+  service: "IAM",
+  policy_set: "CIS",
+  cce_id: "CCE-78910-7",
+  cis_aws_foundations_securityhub: "1.12",
+  benchmark_control: "1.4",
+  benchmark_version: "1.4.0",
+  cis_controls: "[\"1.4\",\"3.3v8\", \"4.3v7\", \"5.4v8\"]",
+  nist: "AC-6(9)"
```
Comment (rshade) [inline security/aws/iam_hwmfa_enabled_for_root/aws_iam_hwmfa_enabled.pt:19]: These need to be updated from the docs.

File: security/aws/iam_hwmfa_enabled_for_root/aws_iam_hwmfa_enabled.pt [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+      end
+    end
+  end
+end
+
+###############################################################################
+# Escalations
+###############################################################################
+
+escalation "report_no_hwmfa" do
```
Comment (rshade) [inline security/aws/iam_hwmfa_enabled_for_root/aws_iam_hwmfa_enabled.pt:174]: We start all escalations with `esc`

**Review by rshade — CHANGES_REQUESTED** [review]

**Comment by XOmniverse** [comment]
@rshade This should be the final correct version based on our testing. Awaiting review.

File: security/aws/iam_hwmfa_enabled_for_root/aws_iam_hwmfa_enabled.pt [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+info(
+  version: "1.0",
+  provider: "AWS",
+  service: "IAM",
+  policy_set: "CIS",
+  cce_id: "CCE-78910-7",
+  cis_aws_foundations_securityhub: "1.12",
+  benchmark_control: "1.4",
+  benchmark_version: "1.4.0",
+  cis_controls: "[\"1.4\",\"3.3v8\", \"4.3v7\", \"5.4v8\"]",
```
Comment (rshade) [inline security/aws/iam_hwmfa_enabled_for_root/aws_iam_hwmfa_enabled.pt:18]: Can you update these?

**Review by rshade — CHANGES_REQUESTED** [review]

File: security/aws/iam_hwmfa_enabled_for_root/aws_iam_hwmfa_enabled.pt (hunk 3 above)
Comment (XOmniverse) [reply security/aws/iam_hwmfa_enabled_for_root/aws_iam_hwmfa_enabled.pt]: > Can you update these?

Done.

**Review by rshade — APPROVED** [review]
LGTM

**Review by rshade — CHANGES_REQUESTED** [review]
Can you add this one and the one we already merged to the root readme?

**Comment by XOmniverse** [comment]
> Can you add this one and the one we already merged to the root readme?

Done

**Review by rshade — APPROVED** [review]

**Review by rshade — APPROVED** [review]
LGTM

**Pull request merged**